docs/API/V1/list.md

```diff
@@ -47,8 +47,8 @@ would have been selected by the other query terms. This can be used with
 with a paginated display or to limit data transfer requirements.
 
 `owner` string \
-The `username` property of a registered user. Only datasets owned by that user
-will be included in the list.
+Select only datasets owned by the specified username. Unless the username
+matches the authenticated user, only "public" datasets can be selected.
```

Comment:
> Unless the username matches the authenticated user

...or the user has admin access?

Reply:
As long as there's a reasonable way to find all the places when it's time to update them.  (Putting a token like "admin" there might make them easy to find....  😉)